tests/schemas/cy/test_language.json

```diff
@@ -55,7 +57,9 @@
                             "type": "Question",
                             "id": "dob-block",
                             "question": {
-                                "description": "",
+                                "description": [
+                                    "dyddiad geni {person_name_possessive}"
```

Comment:
We shouldn't add placeholders to strings without the associated placeholder schema.